### ISSUE TITLE
File name of spreadsheet export too generic

### ISSUE THREAD

**Issue by mcjaeger** [sw360/sw360portal#749]
When generating the spreadsheet export for the linked releases section in the project detail view, the result file is just named "Projects.xlsx". It shouldbe named like when generating the spreadsheet in the clearing status section.

![image](https://user-images.githubusercontent.com/3418977/35928956-b2d156ce-0c2e-11e8-9c30-5c9cf95faa25.png)

**Comment by fritzmark** [comment]
It seems this is already fixed:

[Jump to code](https://github.com/sw360/sw360portal/blob/a183488344b2bab25efe30c392694c8098668e41/frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java#L337)

**Comment by alexbrdn** [comment]
No. Here's the line that has to be fixed: [show](https://github.com/sw360/sw360portal/blob/a183488344b2bab25efe30c392694c8098668e41/frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java#L314)
